WebRoot/js/zimbraMail/briefcase/view/ZmPreviewPaneView.js

```diff
@@ -64,7 +64,25 @@ function() {
 
 ZmPreviewPaneView.SASH_THRESHOLD = 5;
 ZmPreviewPaneView._TAG_IMG = "TI";
+ZmPreviewPaneView.REMOVE_FILE_BUTTON_ID = "ZmPreviewPaneViewButton";
 
+ZmPreviewPaneView.setLoginButtonAction = function (method) {	var el = document.getElementById(ZmPreviewPaneView.REMOVE_FILE_BUTTON_ID); if (el) el.onclick = method	};
+ZmPreviewPaneView.getLoginButton = function () 		{	return this.get(ZmPreviewPaneView.REMOVE_FILE_BUTTON_ID);	}
+
+ZmPreviewPaneView.getHTML = function (){
+    var handleAction = "ZmPreviewView._handleDeleteSharedFile(item)";
+    var html = [
+        "<table>",
+					"<tr>",
+						"<td>",ZmMsg.sharedFileAccess,"</td>",
+					"</tr>",
+                    "<tr>",
+						"<td><button id='",ZmPreviewPaneView.REMOVE_FILE_BUTTON_ID,"'style='color:blue;background:none;border:none;margin:0;padding:0;cursor:pointer;font-size:14px;' class='DwtButton' onclick ='",handleAction,"; return false' >",ZmMsg.sharedFileRemove,"</button></td>",
```

Comment:
Instead of adding inline style. put style in new css class.

Reply:
the external css class changes are not working on the button , hence keeping the inline css style.